File: codegen/results/v002/m3-max-48gb/lmstudio/qwen3.8-27b-gguf-q4km_nothink_hacked/20260830_214823/typescript/001_csv_to_json/solution.ts
import * as fs from 'fs';

// Read the CSV file
const csvContent = fs.readFileSync('input/data.csv', 'utf-8');

// Split into lines
const lines = csvContent.trim().split('\n');

// Skip header
const header = lines[0].split(',');
const rows = lines.slice(1);

// Map header to field names
const result = rows.map(row => {
    const parts = row.split(',');
    return {
        Name: parts[0].trim(),
        Age: parseInt(parts[1].trim(), 10),
        Email: parts[2].trim(),
        Score: parseFloat(parts[3].trim())
    };
});

// Output JSON
console.log(JSON.stringify(result, null, 2));

File: codegen/results/v002/m3-max-48gb/lmstudio/qwen3.8-27b-gguf-q4km_nothink_hacked/20260830_214823/python/001_csv_to_json/solution.py
import csv
import json

def main():
    records = []
    with open('input/data.csv', 'r') as f:
        reader = csv.DictReader(f)
        for row in reader:
            records.append({
                "Name": row['Name'],
                "Age": int(row['Age']),
                "Email": row['Email'],
                "Score": float(row['Score'])
            })
    print(json.dumps(records, indent=2))

if __name__ == '__main__':
    main()